src/main/java/io/openliberty/tools/common/plugins/config/ServerConfigDropinXmlDocument.java

```diff
@@ -15,38 +15,109 @@
  */
 package io.openliberty.tools.common.plugins.config;
 
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
 import javax.xml.parsers.ParserConfigurationException;
 
 import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.w3c.dom.Text;
+import org.xml.sax.SAXException;
 import org.w3c.dom.Element;
 import org.w3c.dom.Comment;
 
 public class ServerConfigDropinXmlDocument extends XmlDocument {
-    
+
     private Element featureManager = null;
 
-    private ServerConfigDropinXmlDocument() {    
+    private ServerConfigDropinXmlDocument() {
     }
-    
+
     public static ServerConfigDropinXmlDocument newInstance() throws ParserConfigurationException {
         ServerConfigDropinXmlDocument configDocument = new ServerConfigDropinXmlDocument();
         configDocument.createDocument("server");
         return configDocument;
     }
 
+    public static ServerConfigDropinXmlDocument newInstance(File f) throws ParserConfigurationException, SAXException, IOException {
```

Comment:
It would be useful to add javadoc comments to describe this method e.g. what is `File f` parameter for

Reply:
👍